CHANGELOG.adoc

```diff
@@ -5,6 +5,7 @@
 === Shapes
 
 - https://github.com/eclipse-syson/syson/issues/1581[#1581] [diagrams] Display inherited ports as border nodes in diagrams.
+- Customize Elasticsearch index to work with SysML metamodel
```

Comment:
I don't have an issue associated to the shape, is it ok?

Reply:
You can create one :)

Reply:
Please use the 2026.1 section of the changelog (there will be one, once you'll rebase the PR on top of `cooldown` branch)